docs/content/v0.20/examples.md

```diff
@@ -1,7 +1,7 @@
 # KubeStellar Usage Examples
 
 **NOTE**: This is unmaintained material that has only been observed to work for the commit tagged as
-`v0.20.0-alpha.1`. CI regularly tests variants of scenarios 1 and 4 that exercise the copy of the repo that they are embedded in (rather than the copy tagged `v0.20.0-alpha.1`), and contributors can run these tests too; see [the e2e tests](../../../test/e2e).
+`v0.20.0-rc1`. CI regularly tests variants of scenarios 1 and 4 that exercise the copy of the repo that they are embedded in (rather than the copy tagged `v0.20.0-rc1`), and contributors can run these tests too; see [the e2e tests](../../../test/e2e).
```

Comment:
Making this edit now is time-traveling. Line 3 says this is a report of testing done in the past. The plan for this text is to keep it a report of the last successfully tested release. It gets updated after that release is created and passes testing.

Reply:
ok, done